DOCSP-41950: Landing page (#150)
mongodb/docs-php-library: source/index.txt

   /indexes
   /monitoring
   /security
   /data-formats
   /compatibility
   /whats-new
   FAQ </faq>
   /reference

Overview
--------

Welcome to the documentation site for the official {+php-library+}.

The {+driver-short+} provides a high-level abstraction around the lower-level
:php:`mongodb extension <mongodb>`.

The ``mongodb`` extension provides a limited API to connect to a MongoDB
database and execute generic commands, queries, and write operations. In
contrast, the {+driver-short+} provides a full-featured API and models client,
database, and collection objects. If you are developing a PHP application with
MongoDB, consider using the {+driver-short+} instead of the extension alone.

Get Started
-----------

Learn how to install the library and extension, establish a connection to MongoDB,
and begin working with data in the :ref:`php-get-started` tutorial.

Connect to MongoDB
------------------

Learn how to create and configure a connection to a MongoDB deployment

> in the :ref:`php-connect` section.

Author: Note to reviewer: link broken until #147 is merged

---------------------

Learn how you can write data to MongoDB in the :ref:`php-write` section.

Transform Your Data with Aggregation
------------------------------------

Learn how to use the {+driver-short+} to perform aggregation operations in the
:ref:`php-aggregation` section.

Optimize Queries with Indexes
-----------------------------

Learn how to work with common types of indexes in the :ref:`php-indexes`
section.

Monitoring
----------

Learn how to monitor change events in the :ref:`php-monitoring`
section.

Secure Your Data
----------------

Learn about ways you can authenticate your application and encrypt your data in
the :ref:`php-security` section.

Specialized Data Formats
------------------------

Learn how to work with specialized data formats and custom types in the
:ref:`php-data-formats` section.

Compatibility
-------------

See compatibility tables showing the recommended {+driver-short+} version to use for
specific PHP and {+mdb-server+} versions in the :ref:`php-compatibility` section.

What's New
----------

Learn about new features and changes in each version in the :ref:`<php-lib-whats-new>`
section.

.. TODO:
 Upgrade {+driver-short+} Versions

>  ---------------------------------

Reviewer: S: add newline under heading

>  Learn what changes you might need to make to your application to upgrade driver versions

Reviewer: ```suggestion
 Learn what changes you must make to your application to upgrade driver versions
```
